.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: node_js
-sudo: false
+services:
+- docker
+sudo: required
 node_js:
 - '4.2'
 script:
 - npm test
+- make docker-build
```

Comment:
Potentially, we could also run the integration tests. They also require encrypted credentials but take a long time to run 30-45 min